compat/src/render.js

```diff
@@ -223,6 +223,25 @@ options._render = function(vnode) {
 	currentComponent = vnode._component;
 };
 
+const oldDiffed = options.diffed;
+/** @type {(vnode: import('./internal').VNode)} */
+options.diffed = function(vnode) {
+	if (oldDiffed) {
+		oldDiffed(vnode);
+	}
+
+	const props = vnode.props;
+	const dom = vnode._dom;
+	if (
+		dom != null &&
+		vnode.type === 'textarea' &&
+		'value' in props &&
+		props.value !== dom.value
+	) {
+		dom.value = props.value ?? '';
```

Comment:
Currently on a phone so can't check: what does this transpile to in our build chain?

Reply:
Here's a screenshot:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/459878/218194671-c002d4f6-dbe5-4bb6-a7d2-85c63f82c3b3.png">

Manually doing `dom.value = props.value == null ? '' : props.value;` saves two bytes so I'll do that